apps/native/app/tsconfig.json

```diff
@@ -1,18 +1,23 @@
 {
-  "extends": "@tsconfig/react-native/tsconfig.json",
+  "extends": "../../../tsconfig.base.json",
   "compilerOptions": {
-    "baseUrl": ".",
-    "strict": true,
-    "paths": {
-      "@ui": ["src/ui"],
-      "@ui/*": ["src/ui/*"],
-      "@island.is/application/types": ["../../libs/application/types/src"],
-      "@island.is/application/types/*": ["../../libs/application/types/src/*"]
-    }
+    "allowSyntheticDefaultImports": true,
+    "isolatedModules": true,
+    "jsx": "react-native",
+    "lib": ["esnext"],
+    "moduleResolution": "node",
```

Comment:
~Let's skip allowSyntheticDefaultImports, moduleResolution and skipLibCheck since they match the base config.